packages/docs-app/src/examples/datetime2-examples/dateRangePicker3Example.tsx

```diff
@@ -131,6 +138,18 @@ export class DateRangePicker3Example extends React.PureComponent<ExampleProps, D
                         label="Single month only"
                         onChange={this.toggleSingleMonth}
                     />
+                    <Tooltip content="Only respected when showing a single month calendar">
+                        <Switch
+                            checked={this.state.showOutsideDays}
+                            className="keep-bottom-margin"
```

Comment:
super nitpick: It looks like the bottom margin here is getting reset back to `0` by some other CSS selector.

<img width="278" alt="Screenshot 2024-08-06 at 6 08 13 PM" src="https://github.com/user-attachments/assets/44da81b7-ed89-43d4-a268-f33873a2d7cb">